conditionals.js

```diff
@@ -200,14 +232,46 @@ function turnOffLight(lightName) {
 /* 
    -------YOUR CODE-----------------------------------------------------------
 */
-function updateLights(somebodyIsHome, theyAreWatchingTV, itIsDarkOutside, theyAreCooking, theyWentToBed) {
-
-    // Write your code here!  You don't need to return anything, just call the given functions
-    // You should be using if else statements and the function arguments
-    // example of turning a light on
-    turnOnLight("livingRoomLight");
-    // example of turning off a light
-    turnOffLight("livingRoomLight");
+function updateLights(
+  somebodyIsHome,
+  theyAreWatchingTV,
+  itIsDarkOutside,
+  theyAreCooking,
+  theyWentToBed
+) {
+  //   If it's dark out, the porch light should be on.  If it's not dark, then it's off.
+  if (itIsDarkOutside) {
+    turnOnLight("frontPorchLight");
+  } else if (!itIsDarkOutside) {
+    turnOffLight("frontPorchLight");
+  }
+  //   If nobody home, then turn all the lights off (except the porch light).
+  if (!somebodyIsHome) {
+    turnOffLight(
```

Comment:
I think this line is where the issue lies. The `turnOffLight` function is only setup to take one parameter (aka light) at a time. so you need to call this function four separate times passing in a different light each time